content/essentials/memos.mdx

```diff
@@ -27,13 +32,15 @@ Whenever `count` undergoes a change, `isEven` will automatically re-run.
 
 While it is possible to use regular functions to mimic the behavior of a memo, `createMemo` offers distinct advantages:
 
-- Unlike a basic derived signal or effects, memos are optimized to execute only _once_ for each change in their dependencies.
+- Unlike a [basic derived signal](/essentials/signals#derived-signals) or [effects](/essentials/effects), memos are optimized to execute only _once_ for each change in their dependencies.
 This significantly reduces computational overhead.
 
-- In scenarios where a signal is updated once but *accessed* multiple times, the memoized function will only execute once and will **cache** its results. 
-This cache means that any subsequent retrieval of the stored result can be accessed _without_ recalculating, shielding the dependents from redundant updates if the value remains consistent.
+- Where a signal is updated once but *accessed* multiple times, the memoized function will only execute once and will **cache** its results.
 
-- With a memos ability to automatically track dependencies, any signal or memo accessed within its function is **tracked**. 
+- If a cached value remains unchanged despite a change in its dependencies (as determined by `===),` it won't trigger subsequent updates. 
+For example, with `setCount(0)`, `isEven()` updates to `true`; however, `setCount(2)` might recompute `isEven()` and will yield `true`, prompting no further updates in components relying on `isEven()`.
```

Comment:
```suggestion
For example, calling `setCount(0)` and then `setCount(2)` will cause the `isEven` memo to recompute both times, but because both evaluations are `true`, the second update will not cause any further updates in components relying on `isEven()`.
```